src/assets/js/design-redhat.js

```diff
@@ -1,12 +1,90 @@
 (function($){
 
+/* ============================== */
+/*  MENU
+/* ============================== */
+
+$(document).ready(function(){
+    var $screen = $("<div class='rh-design-nav-shelf-screen'></div>").appendTo("body"),
+        $nav_shelf = $(".rh-design-nav-shelf"),
+        $toggle_wrapper = $('.rh-design-nav-toggle-wrapper'),
+        $toggle = $toggle_wrapper.find('.btn-toggle');
+    
+    initNav();
```

Comment:
Left it. It didn't work when I tried moving it. But I did remove openNav();

Reply:
I pulled it up locally to test for myself because it shouldn't NOT work. It's all about how files are being read. The JS does get "uglified" and I think what that is doing is reorganizing the JS to be correct regardless of what is in the source code but we care about the source code because other devs may have to read it at some point. When I moved `initNav();` where `openNav();` was, the navigation still worked for me on mobile all the way to desktop. I even confirmed it by looking at the rendered JS file. So I went ahead and pushed the change because the source code should read functions first and THEN the calls for them as I explained in my previous comment (order of operations). I'll close this conversation now.